tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -152,21 +153,31 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 	args = appendIfUnset(args, "--master-volume-size", "48")
 	args = appendIfUnset(args, "--node-count", "4")
 	args = appendIfUnset(args, "--node-volume-size", "48")
-	args = appendIfUnset(args, "--set", adminAccess)
 	args = appendIfUnset(args, "--zones", strings.Join(zones, ","))
 
 	switch d.CloudProvider {
 	case "aws":
 		if isArm {
 			args = appendIfUnset(args, "--master-size", "c6g.large")
+			args = appendIfUnset(args, "--node-size", "c6g.large")
 		} else {
 			args = appendIfUnset(args, "--master-size", "c5.large")
 		}
 	case "gce":
-		args = appendIfUnset(args, "--master-size", "e2-standard-2")
+		if isArm {
+			args = appendIfUnset(args, "--master-size", "t2a-standard-2")
+			args = appendIfUnset(args, "--node-size", "t2a-standard-2")
+		} else {
+			args = appendIfUnset(args, "--master-size", "e2-standard-2")
+			args = appendIfUnset(args, "--node-size", "e2-standard-2")
+		}
 		if d.GCPProject != "" {
 			args = appendIfUnset(args, "--project", d.GCPProject)
 		}
+		// set some sane default e2e testing behaviour on gce
+		args = appendIfUnset(args, "--gce-service-account", "default")
```

Comment:
The Google Service Account that prow runs with doesn't have permissions to make IAM changes on test projects. I can talk to @ameukam about it at a later date.

Sample failure: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/e2e-ci-kubernetes-e2e-cos-gce-serial-canary/1705258496510922752